### DISCUSSION TITLE
🚀 Welcome to GeoHub

### DISCUSSION THREAD

**Discussion by Ali-El-Mikdadi** [Ali-El-Mikdadi/GeoHub#5] [Announcements]
Exciting News! 🌟

GeoHub's official Discussions board is now live! 🌐 Join us to:

Share your ideas 💡
Report bugs 🐛
Discuss features 🚀
Connect with the GeoHub community 💬
Let's build GeoHub together! Dive in and start chatting.

Happy discussing! 🌍✨

The GeoHub Team

**Comment by codemasterlike** [comment]
Thanks the author for great tools, this is one of the best quiz maker platform which is opened source in the industry.
One idea / proposal to enhance it is to include the option of public quiz / Limited access quiz. 
Public quiz: Everyone can access and take the quiz
Limited access: Only students/users who got the PIN/Code, once they fill in the correct PIN/Code, then they can access it.

Really appreciate and wishing you all the best,
